src/andromede/expression/indexing.py

```diff
@@ -110,6 +110,16 @@ def comp_parameter(self, node: ComponentParameterNode) -> IndexingStructure:
             node.component_id, node.name
         )
 
+    def pb_variable(self, node: ProblemVariableNode) -> IndexingStructure:
+        return self.context.get_component_variable_structure(
```

Comment:
Au moment du calcul de l'indexation, aura-t-on déjà des `ProblemVariableNode` et `ProblemParameterNode` ?

Reply:
Indeed it does not really make sense since the indexing will determine if the problem var is time and scenario dependent, so it should not happen, I replaced it by an exception.